src/firefly/js/api/ApiBuild.js

```diff
@@ -174,7 +174,7 @@ export function buildLowlevelAPI() {
         findDispatch(AppDataCntlr),
         findDispatch(DrawLayerCntlr),
         {dispatchAddCell, dispatchRemoveCell, dispatchEnableSpecialViewer},
-        {dispatchAddSaga}
+        {dispatchAddSaga, dispatchAddActionWatcher, }
```

Comment:
This is a good idea.  Much easier for an API user to understand as oppose to generator.